test/integration/features/basic.feature

```diff
@@ -44,6 +44,15 @@ Feature: Basic
     minishift
     """
 
+  Scenario: Persistent volumes should be listed when minishift start
+    When executing "oc get pv --as system:admin -o=name"
+    Then stderr should be empty
+     And exitcode should equal 0
+     And stdout should contain
+     """
+     persistentvolume/pv0001
+     """
```

Comment:
This is tricky as the actual output will contain 100 PVs . @praveenkumar  I guess there should be something in Gherkin to cross check long sequential outputs. Also if we can deploy a simple application which uses PV that would be great.